azavea/fb-gender-survey-dashboard
# Add functional app template

## Overview
Brings in an app template modified to reflect anticipated dependencies
for this project. The environment successfully bootstraps with `setup`
and `server`, though additional changes may be required to adjust
other STRTA scripts.

Notable changes from the core template are:
- Vagrant base box upgraded to Ubuntu 20.04
- Uses Python3 for Ansible and VM-based script execution
 
Connects #1 

## Testing Instructions
After cloning the repository and checking out this branch:
- Run `./scripts/setup` to bootstrap the environment
- Log onto the VM and run the server to see the default CRA app hosted via WDS on http://localhost:3333
  `vagrant ssh && ./scripts/server`
- Check that other scripts (notably, not `cibuild`) are in working order
- Check that the documentation is generally correct
